payu/subcommands/sync_cmd.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+
+# Standard Library
+import argparse
+import os
+
+# Local
+from payu import cli
+from payu.experiment import Experiment
+from payu.laboratory import Laboratory
+import payu.subcommands.args as args
+from payu import fsops
+
+title = 'sync'
+parameters = {'description': 'Sync model output to a remote directory'}
+
+arguments = [args.model, args.config, args.laboratory, args.dir_path,
+             args.sync_restarts, args.sync_ignore_last]
```

Comment:
It will still sync if there is no `enable` flag or if it's set to `False`. The `enable` flag for automatically calling the sync job after archive or collation (if collation is enabled).

Reply:
Great, thanks for the explanation.